### DISCUSSION TITLE
Is there a way to move the close button to the left side?

### DISCUSSION THREAD

**Discussion by u00101010** [tkashkin/Adwaita-for-Steam#13] [Q&A] [answered]
That's where I've been using them for ages, and it's still bugging me that Steam is the only software that doesn't let us change the window button positions. Is there a way to achieve this with this (or any) skin perhaps?

**Comment by tkashkin** [comment, marked as answer]
> Is there a way to achieve this with this skin

This skin has an installer script that has a few window controls presets.
Run `./install.py -p windowcontrols/left` or `./install.py -p windowcontrols/left-all`.
Run `./install.py -h` and `./install.py -l` to see available options.

> ... (or any) skin

Yes, you'll need to change the button positions in a few files and adjust some other controls if they are now covered by the window buttons.
See [window controls patches](https://github.com/tkashkin/Adwaita-for-Steam/tree/master/patches/windowcontrols) for this skin as an example of what you may need to change.
